mec/cmd/mec/main.go

```diff
@@ -47,56 +47,69 @@ var (
 	registryURL    string
 	resyncPeriod   string
 	namespace      string
+
+	mainlog        = log.RegisterScope("main", "Main function", 0)
```

Comment:
nit: it's a matter of taste, but I prefer just using `logger` everywhere

Reply:
We could have that. There are some clashes however. For instance, in the package server there are two loggers, http and worker.